files/common/etc/default/grub.d/override.cfg

```diff
@@ -35,6 +35,12 @@ GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT mitigations=off"
 #
 GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT ipv6.disable=1"
 
+#
+# On Delphix appliance, disks should have 'noop' I/O scheduler since they
+# would be under ZFS control.
+#
+GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT elevator=noop"
```

Comment:
have we considered the potential affects this may have on the IO report card? we run that on "raw" disks, right? so, there's no ZFS IO scheduling to take advantage of there.. I wonder if we would want to use "noop" in that case?

Reply:
Yes, we get better IO Report Card results with 'noop' scheduler. I have results in DLPX-66227 JIRA case.

Reply:
Ah, sorry.. I see that was already considered in the bug. Thanks.